src/feed/filtering.py

```diff
@@ -39,34 +40,27 @@ def _filter_popular(self, request, queryset, view):
 
     def _filter_personalized(self, request, queryset, view):
         user_id = request.query_params.get("user_id")
-
         if user_id:
             user_id = int(user_id)
         elif request.user.is_authenticated:
             user_id = request.user.id
         else:
             return queryset
 
-        try:
-            personalize_client = getattr(view, "personalize_client", None)
-            if not personalize_client:
-                return queryset
-
-            page_size = view.paginator.page_size if hasattr(view, "paginator") else 30
-            filter_param = request.query_params.get("filter", "new-content")
-
-            recommended_ids = personalize_client.get_recommendations_for_user(
-                user_id=str(user_id),
-                filter=filter_param,
-                num_results=page_size * 3,
-            )
+        personalize_feed_service = getattr(view, "personalize_feed_service", None)
+        if not personalize_feed_service:
+            return queryset
 
-            if recommended_ids:
-                return queryset.filter(id__in=recommended_ids)
+        filter_param = request.query_params.get("filter", None)
+        force_new_param = request.query_params.get("force-new-recs", "false")
```

Comment:
Purpose: If user hard-refreshes on frontend the implication is that they want new results. This helps invalidate cache